flow/flow.go

```diff
@@ -135,12 +135,18 @@ func FLowsFromSFlowSample(ft *FlowTable, sample *layers.SFlowFlowSample, probePa
 	for _, rec := range sample.Records {
 
 		/* FIX(safchain): just keeping the raw packet for now */
-		record, ok := rec.(layers.SFlowRawPacketFlowRecord)
-		if !ok {
-			logging.GetLogger().Critical("1st layer is not SFlowRawPacketFlowRecord type")
+		switch rec.(type) {
+		case layers.SFlowRawPacketFlowRecord:
```

Comment:
Ok let's say add a comment like : 

case layers.SFlowRawPacketFlowRecord:
 /* We only support RawPacket from SFlow probe */